client/payment-details/dispute-details/index.tsx

```diff
@@ -0,0 +1,12 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const DisputeDetails: React.FC< Record< string, never > > = () => {
```

Comment:
No need to change, since it will likely change with future work on this component.
I'd go for defining a `React.FC` without the `<>` only because it looks a bit neater for a component without props.
```suggestion
const DisputeDetails: React.FC = () => {
```

Reply:
`til`. I put `{}` and the linter suggested `Record< string, never >`.